src/MongoDB.Analyzer/Core/Poco/PocoDataFiller.cs

```diff
@@ -12,13 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Newtonsoft.Json.Linq;
+
 namespace MongoDB.Analyzer.Core.Poco;
 
 public static class PocoDataFiller
 {
     private const string CollectionNamespace = "System.Collections.Generic";
+    private const string JsonData = "MongoDB.Analyzer.Core.Poco.Data.Data.json";
     private const int MaxDepth = 3;
 
+    private static readonly JObject s_jsonData;
```

Comment:
Lets change `s_jsonData` to more simple data type. To avoid conversions like `((JArray)pair.Value).ToObject<string[]>`.
It also has to be threadsafe for reading (as it's static).

Reply:
Made it a Dictionary<string, string[]>